src/app/components/feed/jh-main-feed.directive.js

```diff
@@ -33,6 +35,41 @@
           Janus.attachMediaStream(video, newVal);
         }
       });
+      
+      $timeout(function(){
+        var mainFeed = document.getElementById('main-feed');
+        mainFeed.addEventListener('dblclick', function() {
+          // if video is in fullscreen mode
+          if (
```

Comment:
Hmmm, I notice that you're making the PR in the `master` branch. So, I must refer to AngularJS instead Angular 2 :sweat_smile::sweat_smile::sweat_smile: